gating/check/run_system_tests.sh

```diff
@@ -9,6 +9,12 @@ set -o pipefail
 
 ## Variables -----------------------------------------------------------------
 
+# The RPC_PRODUCT_RELEASE and RPC_RELEASE need to be brought into scope
+# before running system tests.
+if [[ ${RE_JOB_IMAGE} =~ .*mnaio.* ]]; then
+  source /opt/rpc-openstack/scripts/functions.sh
```

Comment:
is there a variable for the path to /opt/rpc-openstack ?

Reply:
There doesn't need to be, nor do I think there is one. There is https://github.com/rcbops/rpc-openstack/blob/master/gating/check/pre#L14-L22